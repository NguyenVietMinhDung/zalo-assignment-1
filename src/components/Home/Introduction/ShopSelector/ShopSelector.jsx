import React from 'react';
import ReactTooltip from 'react-tooltip';
import './ShopSelector.scss';

const ShopSelector = () => (
  <div className="shop-selector">
    <div data-tip="Bấm vào đây để đổi sang cửa hàng khác" data-place="right">
      Đổi cửa hàng khác &nbsp;
      <i className="lni lni-chevron-down" />
    </div>
    <ReactTooltip />
  </div>
);

export default ShopSelector;
