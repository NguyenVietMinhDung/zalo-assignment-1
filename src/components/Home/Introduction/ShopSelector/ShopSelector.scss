@import '../../../../styles/variables';

.shop-selector {
	width: 200px;

	margin: 15px 0;
	padding: 5px 20px;

	cursor: pointer;

	color: $light-sky-blue;
	border-radius: 50px;
	background-color: $powder-blue;

	font-weight: 500;

	i {
		font-weight: 500;
	}
}
