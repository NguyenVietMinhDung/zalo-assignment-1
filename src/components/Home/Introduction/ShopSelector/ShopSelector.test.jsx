import React from 'react';
import { shallow } from 'enzyme';
import ShopSelector from './ShopSelector';

describe('<ShopSelector />', () => {
  test('renders correctly', () => {
    const component = shallow(<ShopSelector />);
    expect(component).toMatchSnapshot();
  });
});
