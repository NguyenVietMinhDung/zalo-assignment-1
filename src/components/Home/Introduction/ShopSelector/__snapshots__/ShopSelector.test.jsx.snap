// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShopSelector /> renders correctly 1`] = `
<div
  className="shop-selector"
>
  <div
    data-place="right"
    data-tip="Bấm vào đây để đổi sang cửa hàng khác"
  >
    Đổi cửa hàng khác  
    <i
      className="lni lni-chevron-down"
    />
  </div>
  <ReactTooltip
    clickable={false}
    insecure={true}
    resizeHide={true}
    wrapper="div"
  />
</div>
`;
